src/Microsoft.Dnx.Tooling/List/LibraryDependencyFinder.cs

```diff
@@ -4,41 +4,45 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.Extensions.Internal;
-using Microsoft.Dnx.Tooling.Algorithms;
 using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Tooling.Algorithms;
+using Microsoft.Extensions.Internal;
 
 namespace Microsoft.Dnx.Tooling.List
 {
     public class LibraryDependencyFinder
     {
-        public static IGraphNode<LibraryDescription> Build([NotNull] IEnumerable<LibraryDescription> libraries, 
-                                                           [NotNull]Runtime.Project project)
+        public static IGraphNode<LibraryDependency> Build([NotNull] IEnumerable<LibraryDescription> libraries,
+                                                          [NotNull] Runtime.Project project)
         {
-            var root = libraries.FirstOrDefault(p => string.Equals(p.Identity.Name, project.Name));
+            var root = new LibraryDependency
+            {
+                Library = libraries.FirstOrDefault(p => string.Equals(p.Identity.Name, project.Name)),
```

Comment:
I prefer this form that in case we need to add ComparisonOption the code change will be smaller.